packages/vm/src/runBlock.ts

```diff
@@ -151,6 +155,8 @@ export async function runBlock(vm: VM, opts: RunBlockOpts): Promise<RunBlockResu
       // Update the stateRoot cache
       await stateManager.setStateRoot(block.header.stateRoot)
 
+      // Initialize the access witness
+      vm.evm.verkleAccessWitness = new VerkleAccessWitness({ verkleCrypto: vm.evm.verkleCrypto })
```

Comment:
Could we just borrow the `verkleCrypto` from the verkle state manager rather than having to add it as a property on `evm`/`vm`?  Seems like we're adding some unnecessary overhead for instantiating the `vm`/`evm` for this very isolated use case. If you are doing verkle stuff, you already instantiate it in the verkle state manager classes (either directly or indirectly through the `create*StateManager` constructor functions).  Seems like a better user experience to not have to bother instantiating it.
```suggestion
      vm.evm.verkleAccessWitness = new VerkleAccessWitness({ verkleCrypto: vm.statemanager['verkleCrypto'] })
```

Reply:
@gabrocheleau @acolytec3 I think with verkleCrypto we should go over Common here as we do with all our custom and shared crypto, that would be by far the cleanest solution, compared to “borrowing implicitly from somewhere”.

Reply:
Yeah, I think that makes sense to me. Created this issue for it: https://github.com/ethereumjs/ethereumjs-monorepo/issues/3774